full-stack-tests/presentation/src/Utils.ts

```diff
@@ -67,17 +67,34 @@ export function getFieldsByLabel(rootFields: Field[], label: string): Field[] {
   return foundFields;
 }
 
-export async function buildTestIModel(name: string, cb: (builder: IModelBuilder) => void): Promise<IModelConnection> {
-  const outputFile = setupOutputFileLocation(name);
+function createValidIModelFileName(imodelName: string) {
+  return sanitize(imodelName.replace(" ", "-")).toLocaleLowerCase();
```

Comment:
Did we have tests that were producing invalid file names?

Reply:
yes, for example this one: `removes 'Source Information -> Model Source' properties`

Reply:
I'm using full test name as the name of the imodel it generates